data-raw/make-mapping.R

```diff
@@ -154,3 +154,70 @@
+.maps[[.tmpn]][["drug"]] <- c("F11", "F12", "F13", "F14", "F15", "F16", "F18", "F19", "Z503", "Z715", "Z722")
+.maps[[.tmpn]][["panc"]] <- c("K85", "K860", "K861", "K868")
+.maps[[.tmpn]][["endo"]] <- c("E01", "E02", "E03", "E05", "E062", "E063", "E065", "E07", "E163", "E164", "E168", "E169", "E20", "E210", "E212", "E213", "E214", "E215", "E22", "E230", "E232", "E233", "E236", "E237", "E240", "E241", "E243", "E244", "E248", "E249", "E25", "E26", "E27", "E31", "E32", "E345", "E348", "E349")
+.maps[[.tmpn]][["utc"]] <- c("N301", "N302", "N31", "N32", "N35", "N36")
+.maps[[.tmpn]][["tub"]] <- c("A15", "A16", "A17", "A18", "A19", "B90")
+.maps[[.tmpn]][["bone"]] <- c("M80", "M830", "M831", "M832", "M833", "M834", "M835", "M838", "M839", "M85", "M863", "M864", "M865", "M866", "M88")
+.maps[[.tmpn]][["osteounc"]] <- c("M810", "M811", "M815", "M818", "M819")
+.maps[[.tmpn]][["immsys"]] <- c("D80", "D81", "D82", "D83", "D84", "D86", "D89")
+.maps[[.tmpn]][["metab"]] <- c("E70", "E71", "E72", "E74", "E75", "E76", "E77", "E78", "E791", "E798", "E799", "E80", "E83", "E85")
+.maps[[.tmpn]][["mentret"]] <- c("F70", "F71", "F72", "F73", "F78", "F79", "F842", "F843", "F844", "E000", "E001", "E002", "E009", "Q90")
+.maps[[.tmpn]][["cvhep"]] <- c("B18", "B942", "Z225")
+.maps[[.tmpn]][["sleep"]] <- c("F51", "G470", "G471", "G472", "G473")
+.maps[[.tmpn]][["inear"]] <- c("H80", "H81", "H83", "H90", "H910", "H911", "H913", "H918", "H919", "H930", "H931", "H932", "H933")
+.maps[[.tmpn]][["cinfnos"]] <- c("A30", "A31", "A52", "B91", "B92", "B941", "B948", "B949")
+.maps[[.tmpn]][["maln"]] <- c("E40", "E41", "E42", "E43", "E44", "E45", "E46", "E50", "E51", "E52", "E53", "E54", "E55", "E56", "E58", "E59", "E60", "E61", "E63", "E64")
+.maps[[.tmpn]][["ceye"]] <- c("H16", "H181", "H184", "H185", "H186", "H201", "H212", "H301", "H311", "H312", "H313", "H314", "H330", "H332", "H333", "H334", "H335", "H34", "H35", "H43", "H46", "H47", "H49", "H50", "H51", "H530", "H531", "H532", "H533", "H534", "H536", "H538", "H539", "H54", "Q12", "Q13", "Q14", "Q15")
+.maps[[.tmpn]][["cdnos"]] <- c("I119", "I248", "I249", "I250", "I251", "I253", "I254", "I256", "I258", "I259", "I310", "I311", "I421", "I422", "I424")
+.maps[[.tmpn]][["intest"]] <- c("K57", "K592", "K593", "K90")
+.maps[[.tmpn]][["jsd"]] <- c("M07", "M13", "M150", "M151", "M152", "M154", "M158", "M159", "M400", "M402", "M403", "M404", "M405", "M41", "M42", "M43", "M45", "M460", "M461", "M462", "M47", "M480", "M481", "M482", "M485", "M488", "M489", "G950", "G951")
+.maps[[.tmpn]][["flag_comp_diab"]] <- c("I20", "I21", "I22", "I23", "I24", "I25", "I6", "I7", "N03", "N04", "N18", "G603", "G62", "G638", "H35", "H36", "L97")
+.maps[[.tmpn]][["flag_exc_osteo"]] <- c("M80", "S220", "S320", "S52", "S720")
+.maps[[.tmpn]][["flag_exc_hyp"]] <- c("I11", "I12", "I13", "I20", "I21", "I22", "I23", "I24", "I25", "I6", "I70", "I71", "I72", "N03", "N04", "N18")
```

Comment:
Could you please re-arrange these to be in the same order of Supplementary Table 5 in the Stanley and Sarfati paper?

Reply:
Good suggestion, the latest commit has this implemented.

Reply:
Super. Thanks!